crates/astria-cli/src/cli/sequencer.rs

```diff
@@ -29,29 +31,56 @@ pub enum Command {
         #[clap(subcommand)]
         command: BlockHeightCommand,
     },
+    /// Command for sending balance between accounts
+    Transfer(TransferArgs),
 }
 
 #[derive(Debug, Subcommand)]
 pub enum AccountCommand {
     /// Create a new Sequencer account
     Create,
+    Balance(BasicAccountArgs),
+    Nonce(BasicAccountArgs),
 }
 
 #[derive(Debug, Subcommand)]
 pub enum BalanceCommand {
     /// Get the balance of a Sequencer account
-    Get(BalanceGetArgs),
+    Get(BasicAccountArgs),
 }
 
 #[derive(Args, Debug)]
-pub struct BalanceGetArgs {
+pub struct BasicAccountArgs {
     /// The url of the Sequencer node
-    #[clap(long)]
+    #[clap(
+        long,
+        env = "SEQUENCER_URL", 
+        default_value = DEFAULT_SEQUENCER_RPC
+    )]
     pub(crate) sequencer_url: String,
     /// The address of the Sequencer account
     pub(crate) address: SequencerAddressArg,
 }
 
+#[derive(Args, Debug)]
+pub struct TransferArgs {
+    // The address of the Sequencer account to send amount to
+    pub(crate) to_address: SequencerAddressArg,
+    // The amount being sent
+    #[clap(long)]
+    pub(crate) amount: u128,
+    /// The private key of account being sent from
+    #[clap(long, env = "SEQUENCER_PRIVATE_KEY")]
+    pub(crate) private_key: String,
```

Comment:
If this is a clear text private key then please zero it out at drop.

Use the zeroize crate for that.

Reply:
We should definitely come up with a standard here, I agree for services but I question if this is correct behavior for a CLI.

I'm handling this the same as we are handling private keys in CLI rollup deployment.